.github/workflows/release.yml

```diff
@@ -0,0 +1,87 @@
+name: Release packages and images
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+      - edited
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Publish package
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  docker:
+    strategy:
+      matrix:
+        cml: [0]
+        dvc: [1, 2]
+        gpu: [false, true]
+        base: [0, 1]
+        include:
+          - base: 0
+            cuda: 10.1
+            cudnn: 7
+            python: 2.7
+            ubuntu: 18.04
+          - base: 1
+            cuda: 11.0.3
+            cudnn: 8
+            python: 3.8
+            ubuntu: 20.04
```

Comment:
so

`dvcorg/cml:0-dvc-{1,2}-base-{0,1}`

I guess this discussion has been had regarding this tag convention?

Reply:
Discussion would be a bit far-fetched. 😄  Let's say monologue on #217 followed by some reminders [[1][1], [2][2], [3][3], [4][4], [5][5], [6][6], [7][7], [8][8], [9][9], [10][10], [11][11], [12][12], [13][13]]

[1]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1616070220004000
[2]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1616495302003500
[3]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1616703213011200
[4]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1617917520028200
[5]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1618247562035000
[6]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1618331631038700
[7]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1618519097048200
[8]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1618577496051400
[9]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1618866936056300
[10]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1619042757062900
[11]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1619482669073700
[12]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1619649260084000
[13]: https://iterativeai.slack.com/archives/CT0PAR7TQ/p1619740298087000

Reply:
Of course, you're more than welcome to suggest a different convention.

Reply:
I see at least one (logical, not naming) problem:

- `dvcorg/cml:0-dvc-2-base-0` - will anyone really use this?
- `dvcorg/cml:0-dvc-1-base-1` - also any idea how many will use this?

Reply:
* `dvcorg/cml:0-dvc-2-base-0`

Users that use DVC 2 but rely on Python 2.7 and CUDA 10 (like in some Tensorflow <=2.1.0 projects).

Reply:
> Users that use DVC 2 but rely on Python 2.7 and CUDA 10 (like in some Tensorflow <=2.1.0 projects).

Given that `dvc>=2` doesn't install on `py==2.7` (`py3` libs will have to be present) I really doubt there are many people who would even know how to set up such a case on their own systems.

Reply:
> Given that `dvc>=2` doesn't install on `py==2.7` (`py3` libs will have to be present) I really doubt there are many people who would even know how to set up such a case on their own systems.

I thought the same, **but**... DVC comes bundled with [PyInstaller](https://pyinstaller.readthedocs.io/en/stable) and doesn't rely at all on the system Python.

Reply:
Yes, I know, but the point is *how many users will be in this category*?

Reply:
I'm willing to say this is ok. The next point I'd have then is:

`dvcorg/cml:0-dvc-2-base-1` -> `dvcorg/cml:0-dvc2-base1`

let's get rid of some dashes! It looks more readable and quicker to type to me...

Reply:
Not sure, because I don't have access to DVC 1 vs 2 telemetry metrics, if such thing exists.

Reply:
> _let's get rid of some dashes! It looks more readable and quicker to type to me..._

Agreed!